CopperEgg.psm1

```diff
@@ -106,7 +106,7 @@ param(
     $data
   )
   $rtn = ""|select Response,Return
```

Comment:
While making final commit following things needs to be done :
# Remove config.yml file
# Include MSSQL again in config-sample.yml
# Include MSSQL block again in  Start-CopperEggMonitor.ps1
# Change URI to point to api.copperegg.com instaed of api.staging.cuegg.net